release/airflow_scripts.sh

```diff
@@ -100,12 +45,12 @@ function test_command() {
     git config --global user.name "TestRunnerBot"
     git config --global user.email "testrunner@istio.io"
     ls -l    ./githubctl
-    ./githubctl \
-    --token_file="$TOKEN_FILE" \
-    --op=dailyRelQual \
+    ./githubctl                      \
```

Comment:
let's not change the indentation like this. adding a long flag value will mess up all existing indentation.